mlflow/metrics/genai/model_utils.py

```diff
@@ -145,7 +145,7 @@ def _call_llm_provider_api(
 
     payload = {
         k: v
-        for k, v in chat_request.model_dump().items()
+        for k, v in chat_request.model_dump(exclude_none=True).items()
```

Comment:
if exclude_none is not set, some keys may be present which don't belong (e.g. tool_calls in user messages)